src/pages/PushNofiPage/notifi.tsx

```
@@ -0,0 +1,57 @@
+import { Pressable, ScrollView, View } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import Button from '@/components/Button';
+import Header from '@/components/Header';
+import { FILLBOM_COLOR } from '@/constants/color';
+import NoPatients from './components/NoPatients';
+import PatientCardAccepted from './components/PatientCardAccepted';
+import PatientCardPending from './components/PatientCardPending';
+import useGetPatientList from './hooks/useGetPatientList';
+import { patientCardStyles, styles } from './styles';
+
+const pushNofiPage = () => {
```

ESLint

[error] 'pushNofiPage' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)